src/GroupRow.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { memo } from 'react';
+import clsx from 'clsx';
+import { GroupRowRendererProps, CalculatedColumn } from './types';
+import { SELECT_COLUMN_KEY } from './Columns';
+
+function GroupedRow<R, SR>({
+  id,
+  groupKey,
+  viewportColumns,
+  childRows,
+  rowIdx,
+  lastFrozenColumnIndex,
+  top,
+  level,
+  isExpanded,
+  isCellSelected,
+  isRowSelected,
+  groupBy,
+  eventBus,
+  onKeyDown,
+  'aria-setsize': ariaSetSize,
+  'aria-posinset': ariaPosInSet,
+  'aria-rowindex': ariaRowIndex,
+  ...props
+}: GroupRowRendererProps<R, SR>) {
+  // Select is always the first column
+  const idx = viewportColumns[0].key === SELECT_COLUMN_KEY ? level + 1 : level;
+
+  function selectGroup() {
```

Comment:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/treegrid/treegrid-1.html

Check the `Example Usage Options` section. Should we focus on both rows and cells?